.github/workflows/vscode.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
```

Comment:
>on [](http://example.com/codeflow?start=0&length=2)

this pipeline should also work on PRs and merges to main

Reply:
I have changed that can you please take a look.